generators/service-cloudant/templates/swift/instrumentation.swift

```diff
@@ -1,10 +1,10 @@
 import LoggerAPI
+import CloudEnvironment
 import CouchDB
 
-var couchDBClient: CouchDBClient!
-
-func initializeServiceCloudant() throws {
+func initializeServiceCloudant(cloudEnv: CloudEnv) throws -> CouchDBClient {
     // Load credentials for Cloudant/CouchDB using CloudEnvironment
+    let couchDBClient: CouchDBClient
```

Comment:
If you move this declaration below the guard you won't need the type annotation since it would be inferred by the assignment.

Reply:
done